currency/manager_types.go

```diff
@@ -36,7 +36,6 @@ type PairFormat struct {
 	Uppercase bool   `json:"uppercase"`
 	Delimiter string `json:"delimiter,omitempty"`
 	Separator string `json:"separator,omitempty"`
```

Comment:
I think we *can* purge it, bit I'll do that in a separate PR because it's not a super fast slam dunk and I want to approach it separately. Added to my list.